### ISSUE TITLE
Incorrect handling of UTF8 characters on SPARQL HTTP endpoint

### ISSUE THREAD

**Issue by Tomas2D** [openlink/virtuoso-opensource#1026]
Running following query on latest Virtuoso HTTP SPARQL endpoint, no triples are returned, however running following query via JDBC driver  the expected triples are returned.

```
SELECT *
WHERE {
   SERVICE <https://ruian.linked.opendata.cz/sparql> {
    <https://ruian.linked.opendata.cz/zdroj/stavební-objekty/21868701> <https://ruian.linked.opendata.cz/slovník/početBytů> ?flatCount .
   }
}
```

**Comment by HughWilliams** [comment]
Is this a query that was working with a previous VOS build, as I have the same problem with the current `stable/7` build?

And can [recreate against our one of our public endpoints (URIBurner.com)](https://linkeddata.uriburner.com/sparql?default-graph-uri=&query=SELECT+*%0D%0AWHERE+%7B%0D%0A+++SERVICE+%3Chttps%3A%2F%2Fruian.linked.opendata.cz%2Fsparql%3E+%7B%0D%0A++++%3Chttps%3A%2F%2Fruian.linked.opendata.cz%2Fzdroj%2Fstavebn%C3%AD-objekty%2F21868701%3E+%3Chttps%3A%2F%2Fruian.linked.opendata.cz%2Fslovn%C3%ADk%2Fpo%C4%8DetByt%C5%AF%3E+%3FflatCount+.%0D%0A+++%7D%0D%0A%7D&should-sponge=&format=text%2Fhtml&timeout=30000000), whereas it returns results when queried via the "isql" command line tool, i.e., SQL/JDBC, etc.
```pre
SQL> SPARQL SELECT * WHERE {    SERVICE <https://ruian.linked.opendata.cz/sparql> {     <https://ruian.linked.opendata.cz/zdroj/stavební-objekty/21868701> <https://ruian.linked.opendata.cz/slovník/početBytů> ?flatCount .    } };
flatCount
LONG VARCHAR
_______________________________________________________________________________

107

1 Rows. -- 172 msec.
SQL>
```
I suspect it is the wide chars in the IRIs that are not being coded correctly when being passed by the SPARQL endpoint or SPARQL-FED service, as if I run a query of the same type with no wide chars then [it returns results](https://linkeddata.uriburner.com/sparql?default-graph-uri=&query=SELECT+*%0D%0AWHERE+%7B%0D%0A+++SERVICE+%3Chttps%3A%2F%2Fruian.linked.opendata.cz%2Fsparql%3E+%7B%0D%0A++++%3Chttp%3A%2F%2Fwww.openlinksw.com%2Fvirtrdf-data-formats%23sql-integer-uri-fn%3E+%3Chttp%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23type%3E+%3Fo+.%0D%0A+++%7D%0D%0A%7D&should-sponge=&format=text%2Fhtml&timeout=30000000).

We shall look into this ...

**Comment by Tomas2D** [comment]
I did not try this query with the build on stable branch, so I cant tell.

**Comment by pkleef** [comment]
This has been fixed in develop/7 commit id c67527f12746b1c9b6c61b83a85ef0adce364600
